src/test_pixel_gap_cuts.cpp

```diff
@@ -43,11 +44,12 @@ int main(int argc, char** argv) {
     return 1;
   }
   auto compactFile = DETECTOR_PATH + "/" + DETECTOR_CONFIG + ".xml";
-  auto det = &dd4hep::Detector::getInstance();
-  det->fromXML(compactFile);
+  dd4hep::Detector & det = dd4hep::Detector::getInstance();
```

Comment:
This is a deprecated interface, https://github.com/AIDASoft/DD4hep/blob/1962f7d1ace4484efb8886d66a6759a83c5107a3/DDCore/include/DD4hep/Detector.h#L77-L82. Ownership semantics better expressed with `make_unique`, https://github.com/AIDASoft/DD4hep/blob/1962f7d1ace4484efb8886d66a6759a83c5107a3/DDCore/include/DD4hep/Detector.h#L77-L82.

Reply:
The both links are pointing to the same line. can you please point where can I look for the better expression with `make_unique`?

Reply:
Further in the same file; https://github.com/AIDASoft/DD4hep/blob/1962f7d1ace4484efb8886d66a6759a83c5107a3/DDCore/include/DD4hep/Detector.h#L349

Reply:
Thanks a lot. But sorry again another stupid question.
This make_unique method will return me of a type unique_ptr<dd4hep::Detector>, now in order to access
ReadoutGeo, I need a CellIDPoisitionConverter which accepts just dd4hep::Detector. How can use it then?

Reply:
> This is a deprecated interface

Meh, let's let it break when it breaks. This pixel gap test should be moved to the benchmarks, if access to the EICrecon services could be provided.